client/src/views/admin/components/schedule/DeleteScheduleInspector.vue

```diff
@@ -0,0 +1,162 @@
+    >
+      <span class="btn-text">
+        {{ $formatMessage({ id: 'supprimer_la_journnee'}) }}
+      </span>
+    </v-btn>
+    <v-btn
+      color="#DC143C"
+      :disabled="flag === 2"
+      @click="selectTypeOfDelete(2)"
+    >
+      <span class="btn-text">
+        {{ $formatMessage({ id: 'supprimer_la_matinee'}) }}
+      </span>
+    </v-btn>
+    <v-btn
+      color="#DC143C"
+      :disabled="flag === 3"
+      @click="selectTypeOfDelete(3)"
+    >
+      <span class="btn-text">
+        {{ $formatMessage({ id: 'supprimer_l_apres_midi'}) }}
+      </span>
+    </v-btn>
+    <confirm-box
+      v-if="!isCancel"
+      :close-action="cancelAction"
+      :submit-action="deleteInspecteurPlaces"
+    >
+    </confirm-box>
+  </v-card>
+</template>
+
+<script>
+import {
+  getFrenchLuxonFromIso,
+  getFrenchLuxonFromObject,
+} from '@/util'
+
+import { DELETE_INSPECTEUR_PLACES_REQUEST } from '@/store'
+
+import ConfirmBox from '@/components/ConfirmBox.vue'
+
+export default {
+  components: {
+    ConfirmBox,
+  },
+
+  props: {
+    closeDetails: Function,
+    inspecteurId: String,
+    placeInfo: Object,
+  },
+
+  data () {
+    return {
+      isCancel: true,
+      flag: undefined,
+    }
+  },
+
+  methods: {
+    cancelAction () {
+      this.isCancel = true
+      this.flag = undefined
+    },
+
+    selectTypeOfDelete (flag) {
+      this.flag = flag
+      this.isCancel = false
+    },
+
+    getMorningOrAfternoonTimeSlots (placeInfos, status) {
+      if (!status || !placeInfos || !placeInfos.place) {
+        return false
+      }
+      const dateToCompare = getFrenchLuxonFromIso(placeInfos.place.date)
+      if (status === 1) {
+        const startComparatorDate = getFrenchLuxonFromObject({
+          year: dateToCompare.year,
+          month: dateToCompare.month,
+          day: dateToCompare.day,
+          hour: 8,
+          minute: 0,
+        })
+        const endComparatorDate = getFrenchLuxonFromObject({
+          year: dateToCompare.year,
+          month: dateToCompare.month,
+          day: dateToCompare.day,
+          hour: 11,
+          minute: 30,
+        })
+
+        if (dateToCompare >= startComparatorDate && dateToCompare <= endComparatorDate) {
+          return true
+        }
+      }
+      if (status === 2) {
+        const startComparatorDate = getFrenchLuxonFromObject({
+          year: dateToCompare.year,
+          month: dateToCompare.month,
+          day: dateToCompare.day,
+          hour: 13,
+          minute: 30,
+        })
+        const endComparatorDate = getFrenchLuxonFromObject({
+          year: dateToCompare.year,
+          month: dateToCompare.month,
+          day: dateToCompare.day,
+          hour: 15,
+          minute: 30,
+        })
+        if (dateToCompare >= startComparatorDate && dateToCompare <= endComparatorDate) {
+          return true
+        }
+      }
+    },
+
+    async deleteInspecteurPlaces () {
+      let toDispatch
+      if (this.flag === 1) {
```

Comment:
Idem : que veut dire `this.flag === 1` ? Incompréhensible sans passer beaucoup de temps à comprendre tout le contexte.